package.json

```
@@ -13,10 +13,12 @@
     "axios": "^1.9.0",
     "lucide-react": "^0.511.0",
     "prettier": "^3.5.3",
+    "prop-types": "^15.8.1",
```

Biome (1.9.4)

[error] The key prop-types was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)